supplementary_style_guide/glossary_terms_conventions/product_conventions/partner-and-hybrid-cloud-infrastructure.adoc

```diff
@@ -0,0 +1,762 @@
+==== image:images/yes.png[yes] distributed file system (noun)
+*Description*: A file system that presents files from a number of different storage devices, potentially on many different machines and in many different locations, as a single interface to an end user or consuming service.
+
+*Use it*: Yes
+
+[discrete]
+[[distributed-volume]]
+==== image:images/yes.png[yes] distributed volume (noun)
+*Description*: A Gluster volume that distributes data across one or more bricks.
+
+*Use it*: Yes
+
+
+[discrete]
+[[distributed-dispersed-volume]]
+==== image:images/yes.png[yes] distributed-dispersed volume (noun)
+*Description*: A Gluster volume that distributes and erasure codes data across one or more bricks.
+
+*Use it*: Yes
+
+[discrete]
+[[distributed-replicated-volume]]
+==== image:images/yes.png[yes] distributed-dispersed volume (noun)
+*Description*: A Gluster volume that distributes and replicates data across multiple bricks.
+
+*Use it*: Yes
+
+[discrete]
+[[dynamically-provisioned-storage]]
+==== image:images/caution.png[with caution] dynamically provisioned storage (noun)
+*Description*: Generically, a repository of data that changes in size depending on the current demand of the application or the user.
+
+*Use it*: With caution - in OpenShift Container Platform and OpenShift Data Foundation, use "dynamic volume provisioning" to refer to the related StorageClass configuration parameter.
+
+
+[discrete]
+[[encrypted-disk]]
+==== image:images/yes.png[yes] encrypted disk (noun)
+*Description*: A disk containing data that has been encoded for the purpose of ensuring data security.
+
+*Use it*: Yes
+
+
+[discrete]
+[[encryption]]
+==== image:images/yes.png[yes] encryption (noun)
+*Description*: The encoding of data, either at-rest or in-transit, for the purpose of ensuring data security.
+
+*Use it*: Yes
+
+
+[discrete]
+[[ephemeral-storage]]
+==== image:images/yes.png[yes] ephemeral storage (noun)
+*Description*: A temporary storage location that only exists while an individual pod exists, and cannot be shared by multiple pods.
+
+*Use it*: Yes
+
+[discrete]
+[[external-mode]]
+==== image:images/caution.png[with caution] external mode (noun)
+*Description*: Using externally hosted storage services, for example, a deployment of Red Hat OpenShift Container Storage that uses externally hosted Red Hat Ceph Storage (RHCS) clusters to provide storage. This OpenShift Container Storage deployment type is supported for bare metal and user-provisioned VMware environments.
+
+*Use it*: With caution - only in OpenShift Container Storage.
+
+[discrete]
+[[file-storage]]
+==== image:images/yes.png[yes] file storage (noun)
+*Description*: A repository of files and directories organized in a hierarchical way, as distinct from block storage or object storage.
+
+*Use it*: Yes
+
+*See also*: xref:block-storage[block storage], xref:object-storage[object storage]
+
+[discrete]
+[[file-system]]
+==== image:images/yes.png[yes] file system (noun)
```

Comment:
"file system" and "front end"/"front-end" are already in the ISG, can you please remove?